enclave-runtime/src/lib.rs

```diff
@@ -1015,6 +1003,23 @@ where
 	Ok(())
 }
 
+/// Execute a trusted getter on a state and return its value, if available
+///
+/// Also verifies the signature of the trusted getter and return an error
+/// if it's invalid
```

Comment:
I know I am picky, but I prefer dots at the end of a sentence in the documentation so that it is an actual sentence. It looks like more care was given when writing the code. :)

Also, verifies and return does not have the same verb form.

Reply:
I don't think that's too picky - those are good arguments and I'll make the changes. I'm glad my changes are reviewed so thoroughly 😄